README.md

```diff
@@ -95,6 +95,31 @@ variables. There's more detailed documentation in `config.template.yaml`.
 If you don't want to use the config file and use only environment variables,
 set the environment variable `IGNORE_CONFIG_FILE=true`.
 
+It is possible to host Gavel and a website (i.e. an event's website)
```

Comment:
This information seems a bit specific for the README. Could we put it in the [wiki](https://github.com/anishathalye/gavel/wiki) instead?

Reply:
I have a stronger desire to put it in the README than the wiki.
https://github.com/mrvautin/adminMongo has the same type of feature and includes information for changing the base URL in their README.